webpack.config.js

```diff
@@ -2,10 +2,8 @@ require("dotenv").config();
 const webpack = require("webpack");
 const path = require("path");
 const fs = require("fs");
-const CreateFileWebpack = require("create-file-webpack");
```

Comment:
this and the one below were no longer used. noticed this while cleaning out stuff i'd been doing in this file.